solutions/swb-app/src/generateRouter.ts

```diff
@@ -51,6 +52,7 @@ export function generateRouter(apiRouteConfig: ApiRouteConfig): Express {
   // parse application/json
   app.use(express.json());
   app.use(cookieParser());
+  app.use(csurf('none'));
```

Comment:
Is `'none'` necessary? Based on workbench-core/authentication/docs/authenticationMiddleware.md, would passing in no arguments lead to default behavior?

Reply:
Nevermind, I got the the part that default is `'strict'`.